docs/products/kafka/howto/group-requests.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Manage group requests
+limited: true
+---
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+The group requests page allows you to view and track requests made by you and other members of your group for Aiven for Apache Kafka resources.
+These requests can include claims for existing topics or requests to create new topics.
+
+### Key elements
+
+The group requests page displays the following key elements:
+
+- **Topic**: Name of the topic
+- **Type**: Indicates the type of request
+- **Status**: Current status of the request
+  - **Approved**: Request approved and processed
+  - **Declined**: Request declined. The topic is now available for another group to claim
+  - **Deleted**: Request has been deleted by the requester
+  - **Failed**: Request cannot complete the required action (for example, due to
+    Apache Kafka cluster issues or connectivity problems)
+  - **Pending**: Request awaiting approval or decline
+  - **Provisioning**: Request in progress. The topic is being created on the Apache Kafka
+    cluster
+- **Requesting group**: Group that made the request
+- **Requester**: User who submitted the request
+- **Requested on**: Date and time the request was made
+- **Filters**: Filter the list by type, status, requester, and group
+- **Search function**: Search for specific topics by name
+
+### Delete requests
+
+You can only delete requests you have created. You can view requests from other group
+members but cannot delete them.
```

Comment:
You might consider using a note here:

```suggestion
:::note
You can only delete requests you have created. You can view requests from other group
members but cannot delete them.
:::
```

Reply:
I would avoid using note here and since this line is  directly related to the task at hand.